src/sage/schemes/curves/zariski_vankampen.py

```diff
@@ -1378,8 +1379,8 @@ def braid2rels(L):
     br1 = B0.delta()**r * B0(prod(B0(_) for _ in br0_left[1:]))
     cox = prod(F0.gens())
     U0 = [cox**q * (f0 * br1) / cox**q / f0 for f0 in F0.gens()[:-1]]
-    U = [tuple(sign(k1)*(abs(k1) + k) for k1 in _.Tietze()) for _ in U0]
-    pasos = [B.one()] + [_ for _ in reversed(L1)]
+    U = [tuple(sign(k1) * (abs(k1) + k) for k1 in _.Tietze()) for _ in U0]
+    pasos = [B.one()] + list(reversed(L1))
```

Comment:
```diff
- br1 = B0.delta()**r * B0(prod(B0(_) for _ in br0_left[1:]))
+ br1 = B0.delta()**r * prod(map(B0, br0_left[1:]), B0.one())
```

Also, do you mind renaming `_` to say `br` or something sensible? Variable names shouldn't start with underscores if they are used.